app/forms/totp_setup_form.rb

```diff
@@ -0,0 +1,27 @@
+class TotpSetupForm
```

Comment:
I thought about that, until I realized the controller spec serves as an integration test for the form. Do you think we need a unit test as well?